packages/next/lib/recursive-copy.ts

```diff
@@ -14,7 +14,7 @@ export async function recursiveCopy(
   source: string,
   dest: string,
   {
-    concurrency = 255,
+    concurrency = 5,
```

Comment:
I'm guessing this is going to have negative consequences for the benchmark.

Reply:
The benchmark appears to be crashing 🤔 looks like we don't test copying when the files already exist since the test removes the [`testDir`](https://github.com/zeit/next.js/blob/ba5e6943fa33bd29e5e7261de818faf8eb191943/test/unit/recursive-copy.test.js#L26) after each run